contracts/cw-proposal-single/src/tests.rs

```diff
@@ -72,6 +72,24 @@ fn cw20_staked_balances_voting() -> Box<dyn Contract<Empty>> {
     Box::new(contract)
 }
 
+fn cw721_base() -> Box<dyn Contract<Empty>> {
```

Comment:
nit: i see a future where we want to have a `tests` dir and seperate via different voting power modules, e.g. cw20, cw4, cw721

Reply:
should of been more clear here: this is probably future work